ranking/views.py

```diff
@@ -13,3 +14,42 @@ class RankingView(LoginRequiredMixin, SingleTableView):
     def get_queryset(self):
         return ChallengeUser.objects.filter(success=True).values('user__username')\
             .annotate(count=Count('*'), time=Max('last_try')).order_by('-count', 'time')
+
+
+def get_chart(request):
```

Comment:
This endpoint is not secure. I would recommend to use a class that inherits the mixins of RankingView, then use the method get to put this code. (It works the same but with the mixins it has build in security).